### PR TITLE
[rfc-implementation] Callsystems

### PR DESCRIPTION
**Pull request by daniwasonline**
This is the implementation for #77.

### CODE REVIEW

File: src/callsystems/legacy/index.js
Comment (artifishvr) [inline src/callsystems/legacy/index.js:53]: ok i know we already do this but why not use a system role instead?

Comment (daniwasonline) [reply src/callsystems/legacy/index.js:53]: i just moved everything over from the old legacy backend, but the 8B variant of LLaMA doesn't have a very good context window

**Review by artifishvr — APPROVED** [review]

**Comment by daniwasonline** [comment]
@artifishvr I'm so sorry for making you review this so many times! I keep on intending to push Callsystems into master, but planning out Integrations requires so many moving pieces that I keep on having to iterate on this standard to add the required facilities in order to build it.

**Comment by daniwasonline** [comment]
Callsystems has reached API stability and will enter an API freeze until the RFC implementation is merged into `main` (pending final approval).

Read more: https://github.com/spongedsc/pathways/issues/77#issuecomment-2134324614

**Comment by Butterroach** [comment]
oh god v3.0 already

**Comment by daniwasonline** [comment]
> oh god v3.0 already

we're probably gonna have a few more minor releases in the 2.x line because a) feel like it's too soon and b) we can do a few more things that don't require breaking changes in 2.x

**Comment by artifishvr** [comment]
> I'm so sorry for making you review this so many times!

image
/j

File: src/lib/callsystems/std/managers/cuts.js
Comment (artifishvr) [inline src/lib/callsystems/std/managers/cuts.js:88]: is this formatted? why is the closign stuff so far over?

Comment (daniwasonline) [reply src/lib/callsystems/std/managers/cuts.js:88]: there is a lot of nesting in this function, it looks really confusing but i swear this was prettier

**Comment by daniwasonline** [comment]
> > I'm so sorry for making you review this so many times!
> 
> 
> 
> image
> 
> /j

WJTJERHSNCJFKGNDNFKSKDNF IM SORRYYYYY

**Review by artifishvr — APPROVED** [review]
looks good to me! (i am bad at review and this pr is making github lag)

**Comment by daniwasonline** [comment]
> looks good to me! (i am bad at review and this pr is making github lag)

you haven't seen a real commit yet

**Comment by daniwasonline** [comment]
(Don't delete rfc/callsystems for now, potentially may come back into usage later)

**Pull request merged**
